Gemfile

```diff
@@ -4,13 +4,13 @@ gemspec
 
 # for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
 gem 'gettext', '>= 3.1.3', '< 4.0.0'
+gem 'rake', '~> 10.1.0'
```

Comment:
Please use the latest version. Version 10 has known vulnerabilities and is incompatible with Ruby 3.
```suggestion
gem 'rake', '~> 13.0'
```